### PR TITLE
Feature/69/styling

### PR DESCRIPTION
**Pull request by denniskalpedis**
closes ticket #69 and #66 

Styled and added social icons to footer.

### CODE REVIEW

File: frontend/src/components/enroll/enroll.jsx [hunk 1]
```diff
@@ -16,29 +17,32 @@ class Enroll extends Component {
...
             <Markdown source={this.state.text} />
             
-            <div id="contact-form-900">
+            <div class='jumbotron'>
                 <form action="/" method="post" class="contact-form commentsblock">
 
                     <div class="grunion-field-wrap grunion-field-name-wrap">
-                        <label for="g900-name" class="grunion-field-label name">Name<span>(required)</span></label>
-                        <input type="text" name="g900-name" id="g900-name" value="" class="name" required="" aria-required="true" />
+                        <label for="g900-name" class='col-md-2'>Name<span> (required)</span></label>
```
Comment (mkirlin) [inline frontend/src/components/enroll/enroll.jsx]: I know it's getting repetitive, but go ahead and strip out the old CSS & names here too. I'm v tired of g-whatever.

File: frontend/src/router.js [hunk 2]
```diff
@@ -4,29 +4,34 @@ import { Provider } from "react-redux";
...
+import Home from "./components/Home";
 import DemoPage from "./components/DemoPage";
 import NotFound from "./components/NotFound";
 import Login from "./components/Login";
 import Admin from "./pages/admin";
 import Footer from "./components/footer";
 import Contact from "./components/contact";
 import About from "./components/about";
-import Enroll from "./components/enroll";
+import Enroll from "./components/enroll/enroll";
```
Comment (mkirlin) [inline frontend/src/router.js]: Why is this double-nested?

Comment (denniskalpedis) [reply frontend/src/router.js]: fixed

File: frontend/src/components/enroll/enroll.jsx (hunk 1 above)
Comment (denniskalpedis) [reply frontend/src/components/enroll/enroll.jsx]: fixed

File: frontend/src/components/enroll/enroll.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
+.messageInput {
```
[the remark this answers is not in the document]
Comment (denniskalpedis) [reply frontend/src/components/enroll/enroll.css]: fixed

File: frontend/src/components/contact.jsx [hunk 4]
```diff
@@ -19,25 +19,27 @@ class Contact extends Component {
                 <div id="contact-form-12">
                     <form action="/" method="post" class="contact-form commentsblock">
 
-                        <div class="grunion-field-wrap grunion-field-name-wrap">
-                            <label for="g12-name" class="grunion-field-label name">Name<span>(required)</span></label>
-                            <input type="text" name="g12-name" id="g12-name" value="" class="name" required="" aria-required="true" />
-                        </div>
+                    <div class="grunion-field-wrap grunion-field-name-wrap">
```
[the remark this answers is not in the document]
Comment (denniskalpedis) [reply frontend/src/components/contact.jsx]: fixed

File: frontend/src/components/ambassadors.jsx [hunk 5]
```diff
@@ -15,37 +15,45 @@ class Ambassadors extends Component {
```
[the remark this answers is not in the document]
Comment (denniskalpedis) [reply frontend/src/components/ambassadors.jsx]: fixed

File: frontend/src/components/App.css [hunk 6]
```diff
@@ -26,4 +26,8 @@
     margin: 20px;
```
[the remark this answers is not in the document]
Comment (denniskalpedis) [reply frontend/src/components/App.css]: renamed back to app.jsx

File: frontend/src/components/Header/index.js [hunk 7]
```diff
@@ -29,7 +23,7 @@ class Header extends Component {
                 Donate
               </NavLink>
             </li>
-            <li className="nav-item">
+            <li className="nav-item" >
```
[the remark this answers is not in the document]
Comment (denniskalpedis) [reply frontend/src/components/Header/index.js]: fixed

File: frontend/src/components/Carousel.jsx [hunk 8]
```diff
@@ -0,0 +1,23 @@
+import React, { Component } from "react";
```
[the remark this answers is not in the document]
Comment (denniskalpedis) [reply frontend/src/components/Carousel.jsx]: it doesn't work...so no

File: frontend/src/components/Carousel.jsx [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+
+class Carousel extends Component {
+    render() {
+        return (
+    <div id="carouselExampleSlidesOnly" className="carousel slide" data-ride="carousel">
```
[the remark this answers is not in the document]
Comment (denniskalpedis) [reply frontend/src/components/Carousel.jsx]: fixed

File: frontend/src/components/Carousel.jsx [hunk 10]
```diff
@@ -0,0 +1,48 @@
+import React, { Component } from "react";
+import { Carousel } from 'react-bootstrap';
+
+class CarouselComponent extends Component {
+    render() {
+        return (
+<Carousel>
```
Comment (mkirlin) [inline frontend/src/components/Carousel.jsx]: This needs to be formatted/indented properly.

File: frontend/src/components/Carousel.jsx (hunk 8 above)
[the remark this answers is not in the document]
Comment (mkirlin) [reply frontend/src/components/Carousel.jsx]: Soooo...?

File: frontend/src/components/Header/index.jsx [hunk 11]
```diff
@@ -56,7 +50,7 @@ class Header extends Component {
                 Enroll Now
               </NavLink>
             </li>
-            <li className="nav-item">
+            <li className="nav-item" class="nav-item">
```
Comment (mkirlin) [inline frontend/src/components/Header/index.jsx]: Pretty sure this `class` tag is redundant.

File: frontend/src/router.js [hunk 12]
```diff
@@ -4,7 +4,7 @@ import { Provider } from "react-redux";
 import { createStore, applyMiddleware } from "redux";
 import fetchData from "./reducers";
 import thunk from "redux-thunk";
-import App from "./components/App";
+import Home from "./components/App";
```
Comment (mkirlin) [inline frontend/src/router.js]: This component file still needs to be renamed.

File: frontend/src/components/Carousel.jsx (hunk 8 above)
[the remark this answers is not in the document]
Comment (ShawnArd) [reply frontend/src/components/Carousel.jsx]: image

**Pull request merged**